lang/cs/Org.Apache.REEF.Common.Tests/metrics/MetricTestUtils.cs

```diff
@@ -104,6 +105,16 @@ public IMetricsCollector EndRecord()
                 throw new NotImplementedException();
             }
 
+            public IMetricsRecord GetRecord()
+            {
+                throw new NotImplementedException();
```

Comment:
I mean this test utility doesn't test potential use of these unimplemented methods by other classes